harmony-kotlin/src/androidMain/kotlin/com/harmony/kotlin/common/logger/BugfenderAndroidLogger.kt

```diff
@@ -5,9 +5,11 @@ import android.os.Build
 import com.bugfender.sdk.Bugfender
 import java.util.regex.Pattern
 
-open class BugfenderLogger(context: Context,
-                           applicationKey: String,
-                           isDebug: Boolean) : Logger {
+open class BugfenderAndroidLogger(
```

Comment:
@franmontiel I changed the name of the class to do reference to Android.

Reply:
You can keep the previous name. Take into account that this class is only visible for Android. There will be no clash between this and any other BugfenderLogger on iosMain